docs/source/data_analysis/HPC-module-SLEAP.md

```diff
@@ -0,0 +1,610 @@
+# Use the SLEAP module on the HPC cluster
+
+```{include} ../_static/swc-wiki-warning.md
+```
+
+```{include} ../_static/code-blocks-note.md
+```
+
+## Abbreviations
+| Acronym | Meaning                                      |
+| ------- | -------------------------------------------- |
+| SLEAP   | Social LEAP Estimates Animal Poses           |
```

Comment:
Maybe add a link here for both SLEAP and SLURM? Expanding the acronym doesn't help much.